virtinst/guest.py

```diff
@@ -632,7 +632,12 @@ def lookup_domcaps(self):
         def _compare_machine(domcaps):
             if self.os.machine == domcaps.machine:
                 return True
-            capsinfo = self.lookup_capsinfo()
+            try:
+                capsinfo = self.lookup_capsinfo()
+            except Exception:
+                log.exception("Error fetching machine list for alias "
+                                  "resolution, assuming mismatch");
```

Comment:
This is not correctly indented.

Reply:
Hi Laszlo,

Cannot argue with that as it is indeed consistent. Before this comment I only checked few random multiple line strings where it was indented differently.

So let's keep it like it is now.